src/Controls/src/Core/Compatibility/Handlers/ListView/Windows/ListViewStyles.xaml

```diff
@@ -147,14 +149,73 @@
         </platform:EntryCellTextBox>
     </DataTemplate>
 
+    <CornerRadius x:Key="ListViewItemCornerRadius">0</CornerRadius>
+
+    <Style x:Key="DefaultListViewItemStyle" TargetType="ListViewItem">
```

Comment:
Do we need this style?  Is this just copying over the `XF` style so that we stay consistent? Or is there something specific it's addressing?

The padding/cornerradius issues I think are taken care of by these changes

https://github.com/dotnet/maui/pull/8584/files#diff-e1348fd31aa80c506e8dae2220ea037e3bc88003ab31c6ebec1d6beb46b64f43R217-R218

and this one
```XAML
<CornerRadius x:Key="ListViewItemCornerRadius">0</CornerRadius>
```

And then maybe this one :-)
```XAML
<x:Boolean x:Key='ListViewItemSelectionIndicatorVisualEnabled'>False</x:Boolean>
```

Though I do waffle a bit if it's just easier to copy the XF one. 
But maybe it's enough just to set a couple more keys